decidim-process-extended/app/views/decidim/participatory_processes/admin/participatory_processes/_proposal_component_alert.html.erb

```diff
@@ -8,11 +8,11 @@
       proposal_component = current_participatory_process.components.find_by(manifest_name: 'proposals')
     end
 
-    proposal_component.nil? || !proposal_component.published?
+    !proposal_component.nil? && !proposal_component&.published?
```

Comment:
this is doing the contrary of what the method name says.
We can also extract all this block into a helper

Reply:
I try to extract into a helper but I have some problems... can we check it together?